cli/dcoscli/data/help/marathon.txt

```diff
@@ -132,6 +135,9 @@ Positional Arguments:
         https://mesosphere.github.io/marathon/docs/rest-api.html#post-v2-groups.
     <instances>
         The number of instances.
+    <pod-resource>
+        Path to a file or HTTP(S) URL that contains the pod's JSON definition.
+        If omitted, the definition is read from stdin.
```

Comment:
The `app add` and `group add` commands have a link to API docs, but that link doesn't exist yet for pods. The Marathon team is also rewriting their docs in RAML, so I'm not sure what the new URL will be.